clarify SL test/query language (#3945)
fishtown-analytics/docs.getdbt.com: website/snippets/_sl-test-and-query-metrics.md

> :::important Testing and querying metrics in the dbt Cloud IDE not yet supported

Reviewer: this is great! thank you for making this so clear

> For public beta, testing or querying metrics in the dbt Cloud IDE isn't supported yet, however support is coming soon. 

Reviewer 2: It might read better if it's more concise. 
```suggestion
Support for testing or querying metrics in the dbt Cloud IDE is not available in the current beta but is coming soon. 
```

> You can use the **Preview** or **Compile** buttons in the IDE to run semantic validations and make sure your metrics are defined. And you can [dynamically query metrics](#connect-and-query-api) with integrated tools on a dbt Cloud [Team or Enterprise](https://www.getdbt.com/pricing/) plan using the [Semantic Layer API](/docs/dbt-cloud-apis/sl-api-overview).

Reviewer 2: ```suggestion
You can use the **Preview** or **Compile** buttons in the IDE to run semantic validations and make sure your metrics are defined. You can [dynamically query metrics](#connect-and-query-api) with integrated tools on a dbt Cloud [Team or Enterprise](https://www.getdbt.com/pricing/) plan using the [Semantic Layer API](/docs/dbt-cloud-apis/sl-api-overview).
```

> You can currently only test metrics locally using the MetricFlow CLI. Alternatively, you can test using SQL client tools like DataGrip, DBeaver, or RazorSQL. 

Reviewer 2: ```suggestion
Currently, you can test metrics locally using the MetricFlow CLI. dbt Cloud IDE support is coming soon. Alternatively, you can test using SQL client tools like DataGrip, DBeaver, or RazorSQL. 
```

:::

This section will explain how you can test and query metrics locally using the MetricFlow CLI (dbt Cloud IDE support coming soon).

Before you begin, you'll need to install the [MetricFlow CLI](/docs/build/metricflow-cli) package and make sure you run at least one model.
### Install MetricFlow

> import InstallMetricFlow from '/snippets/_sl-install-metricflow.md';

Reviewer: Import
